src/_guides/language/type-system.md

```diff
@@ -475,13 +476,14 @@ which is disallowed from non-`dynamic` types such as `Animal`.
   in the [analysis options file.][analysis]
 {{site.alert.end}}
 
-To make this code pass static analysis, 
-use an explicit cast, 
-which might fail at runtime.
+To make this type of code pass static analysis, 
+you can use an explicit cast, 
+but it might fail at runtime.
 
 <?code-excerpt "lib/strong_analysis.dart (generic-type-assignment-implied-cast)" replace="/as.*(?=;)/[!$&!]/g"?>
 {% prettify dart tag=pre+code %}
-List<Cat> myCats = <Animal>[] [!as List<Cat>!];
+List<Animal> myAnimals = ...
+List<Cat> myCats = myAnimals [!as List<Cat>!];
 {% endprettify %}
```

Comment:
```suggestion
List<Animal> myAnimals = ...
List<Cat> myCats = myAnimals [!as List<Cat>!];
{% endprettify %}

An explicit cast might still fail at runtime, though,
depending on the actual value of the casted list (`myAnimals`).
```

- "why" this might fail is unclear, should be elaborated on a little more, basically how you said it in the PR description
- the fact that it might still fail at run time is kind of an kind of an aside from the point of the section (generic type, downcasting) and from the code itself (passing static analysis with implicit downcast), so it can come last